packages/pds/src/services/feed/index.ts

```diff
@@ -0,0 +1,256 @@
+import { sql } from 'kysely'
+import * as common from '@atproto/common'
+import Database from '../../db'
+import { countAll } from '../../db/util'
+import { ImageUriBuilder } from '../../image/uri'
+import { Presented as PresentedImage } from '../../lexicon/types/app/bsky/embed/images'
+import { View as PostView } from '../../lexicon/types/app/bsky/feed/post'
+import { ActorViewMap, FeedEmbeds, PostInfoMap, FeedItemType } from './types'
+
+export * from './types'
+
+export class FeedService {
+  constructor(public db: Database, public imgUriBuilder: ImageUriBuilder) {}
+
+  static creator() {
+    return (db: Database, imgUriBuilder: ImageUriBuilder) =>
```

Comment:
If you wanted to, you could probably hoist this up like so:
```suggestion
  static creator(imgUriBuilder: ImageUriBuilder) {
    return (db: Database) =>
```

Reply:
oh i like that 👍 
didn't see til not that you'd done that with messageQueue & blobstore